build.sbt

```diff
@@ -10,7 +10,7 @@ libraryDependencies ++= Seq(
   jdbc,
   evolutions,
   "com.adrianhurt" %% "play-bootstrap" % "1.0-P25-B3",
-  "com.typesafe.play" %% "anorm" % "2.5.1",
+  "com.typesafe.play" %% "anorm" % "2.5.0",
```

Comment:
@wsargent due to resolution priority when a simple name (there `NAME`) can be derived several qualified names in the same resultset (there `COMPUTER.NAME` and `COMPANY.NAME`).

See https://github.com/playframework/anorm/issues/102